eng/pipelines/libraries/base-job.yml

```diff
@@ -74,7 +74,7 @@ jobs:
         - ${{ if ne(parameters.liveCoreClrBuildConfig, '') }}:
           - _coreClrDownloadPath: '$(Build.SourcesDirectory)/artifacts/transport/coreclr'
           - _coreClrArtifactName: 'CoreCLRProduct_${{ parameters.osGroup }}${{ parameters.osSubgroup }}_${{ parameters.archType }}_${{ parameters.liveCoreClrBuildConfig }}'
-          - _coreClrOverridePathArg: ' /p:CoreCLROverridePath=$(_coreClrDownloadPath)'
+          - _coreClrOverridePathArg: ' /p:CoreCLRArtifactsPath=$(_coreClrDownloadPath)'
```

Comment:
Nit: we can clean-up later, but it might be worth to change the variable name as well to be consistent in between MSBuild property and variable name.